jetty-security/src/main/java/org/eclipse/jetty/security/ConstraintSecurityHandler.java

```diff
@@ -741,14 +741,9 @@ public boolean checkPathsWithUncoveredHttpMethods()
         Set<String> paths = getPathsWithUncoveredHttpMethods();
         if (paths != null && !paths.isEmpty())
         {
-            ContextHandler.Context currentContext = ContextHandler.getCurrentContext();
-
-            for (String p : paths)
-            {
-                LOG.warn("{} has uncovered http methods for path: {}", currentContext, p);
-            }
-            if (LOG.isDebugEnabled())
-                LOG.debug("{} has uncovered http methods", currentContext, new Throwable());
+            LOG.warn("{} has paths with uncovered http methods: [{}]",
```

Comment:
The log warn message is the wrong way around, it is the _uncovered paths_ not _methods_, so it should be:

```suggestion
            LOG.warn("{} has uncovered http methods for paths: [{}]",
```

Reply:
Yet the method name is `checkPathsWithUncoveredHttpMethods` ??

Reply:
Correct, we are reporting the uncovered _paths_ not the uncovered _methods_.

Reply:
Confusing.

* "has paths with uncovered http methods" - identifies paths, that has uncovered http methods
* "has uncovered http methods for paths" - identifies methods, that are not covered by the paths

Perhaps there's a 3rd choice that's less confusing.  What does "uncovered http method" mean?

Reply:
Agree with @joakime here, the method name suggests the other way around.
Also, do we really want to use WARN? I don't think nothing happens if we hit this case, and we don't want to clog log files if repeated requests for this case happens?

Reply:
The name of the method is "checkPathsWithUncoveredMethods" - ie we are checking the _paths_ for which there are uncovered methods.  The logging is reporting the _paths_, not the methods - it would be incorrect to imply that we are reporting methods!

Furthermore, this warning is required by the servlet specification: your choice if you want to make it INFO instead of WARN, but the spec _requires_ that we produce this warning at deployment time.

Reply:
Is this the section of the spec this covers?

https://github.com/eclipse-ee4j/servlet-api/blob/5.0.0-RELEASE/spec/src/main/asciidoc/servlet-spec-body.adoc#handling-uncovered-http-methods

Reply:
Yes, this is the section. Please see comment by @gregw - we can enumerate the paths that are uncovered but not the methods.